Add plugin: 复制图文 (Copy Image Text)

msgk239 opened the pull request:
# I am submitting a new Community Plugin

## Repo URL
<!--- Paste a link to your repo here for easy access -->
https://github.com/msgk239/obsidian-copy-image-text

## Release Checklist
- [ ] I have tested the plugin on
  - [ x ]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [ ] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [ x ] `main.js`
  - [ x ] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [ x ] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [ x ] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [ x ] My README.md describes the plugin's purpose and provides clear usage instructions.
- [ x ] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [ x ] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [ x ] I have added a license in the LICENSE file.
- [ x ] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
["permissions": ["clipboard-write"]](https://github.com/msgk239/obsidian-copy-image-text/blob/e84ce324ba2e2d53bbc1eef536e17764cf3170f2/manifest.json#L9)
This is not a thing, remove this line.

[arrayBufferToBase64(buffer: ArrayBuffer): Promise<string> {](https://github.com/msgk239/obsidian-copy-image-text/blob/e84ce324ba2e2d53bbc1eef536e17764cf3170f2/main.ts#L103-L114)
That is already a function in the Obsidian API, use that.

msgk239 commented:
@joethei 

Thank you for your review. I have made the following changes:
1. Removed the "permissions": ["clipboard-write"] line from manifest.json
2. Replaced the custom arrayBufferToBase64 implementation with Obsidian's built-in API function

The changes have been committed and the version has been updated from 1.0.5 to 1.0.6.

Please review again. Thank you for your feedback!
